CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Added
+- Airplay and Picture in Picture toggle buttons for the `modernSmallScreenUI`
```

Comment:
As Airplay and picture-in-picture are unrelated I'd go for two changelog entries.